src/python/engagement-creator/src/engagement-creator.py

```diff
@@ -131,6 +130,7 @@ def recalculate_score(lens: LensView) -> int:
 
 
 def set_score(client: GraphClient, uid: str, new_score: int, txn: Any = None) -> None:
```

Comment:
why the comment instead of just deleting?
cuz like
its really odd
that these fns are just sitting there

Reply:
They could totally be dead tbh